### ISSUE TITLE
Low Image Contrast on Settings Page

### ISSUE THREAD

**Issue by Shiwal25** [OneBusAway/onebusaway-android#1419]
The back arrow icon on the Settings screen has insufficient contrast against its green background. This was flagged by the Google Accessibility Scanner:
Current contrast ratio: 2.78:1
Minimum required : 3.0:1

Screenshot:
![Image](https://github.com/user-attachments/assets/3b12723a-31c8-4abb-92d1-720522f9d500)

**Comment by Shiwal25** [comment]
@amrhossamdev assign me the issue

**Comment by Shiwal25** [comment]
Realised it's related to the brand of the app, so I don't think the need to address at this time.
